addons/im_livechat/static/src/public_models/public_livechat_global_notification_handler.js

```diff
@@ -101,5 +104,8 @@ registerModel({
             identifying: true,
             inverse: 'notificationHandler',
         }),
+        eventTypesDependingOnChatWindow: attr({
+            default: ['mail.message/insert', 'mail.channel/new_message', 'mail.channel.member/typing_status'],
+        }),
```

Comment:
I would not introduce this field, and just guard each handling of the notification that are problematic.